db/migrate/20170421193150_add_embedded_methods_to_miq_ae_method.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmbeddedMethodsToMiqAeMethod < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_ae_methods, :embedded_methods, :text, :array => true
```

Comment:
@bdunne @Fryguy 
For existing records #embedded_methods will return a nil but if we set the default value (:default => []) in the add_column field #embedded_methods will return a []. 
That means in the code we don't have to check for nil values.
So my question is along with the model **default_value_for** should we set the add_column default.